Simulations/README.md

```diff
@@ -234,12 +234,13 @@ The set of simulations is as follows
  - N_LSS_WAVE_GUESS
  - N_LSS_DIST_SOL
  - LM_DIST_SOL
- - ATM_LINE_CAT
+ - ATM_PROFILE
```

Comment:
`ATM_LINE_CAT` should still be there too. The problem that was solved in https://github.com/AstarVienna/METIS_Simulations/pull/93 was that both the `ATM_PROFILE` and the `ATM_LINE_CAT` were written to `ATM_LINE_CAT`. Both now have their own file.